driver/driver.go

```diff
@@ -89,3 +92,26 @@ func Boot(ctx, clientContext context.Context, d Driver, pw progress.Writer) (*cl
 		return c, nil
 	}
 }
+
+func HistoryAPISupported(ctx context.Context, c *client.Client) (res bool) {
+	res = true
+	checkErrF := func(err error) {
+		if s, ok := grpcerrors.AsGRPCStatus(err); ok {
+			if s.Code() == codes.Unimplemented {
+				res = false
+			}
+		}
+	}
+	cl, err := c.ControlClient().ListenBuildHistory(ctx, &controlapi.BuildHistoryRequest{
+		ActiveOnly: true,
+		Ref:        "0123456789-abcdefghijklmn", // dummy ref to check if the server supports the API
```

Comment:
*super nitpick*: can we set something that contains the string `buildx-dummy-ref` or similar, so it's ultra-clear if this ends up in a buildx/buildkit error report?